setup.cfg

```diff
@@ -48,7 +48,8 @@ addopts = --reuse-db --randomly-seed=1521188767 --randomly-dont-reorganize
 env=
     DJANGO_OIDC_USERNAME_CLAIM=sub
 filterwarnings =
-    error::DeprecationWarning
+    # comment is needed beceause of https://github.com/mozilla/mozilla-django-oidc/pull/371
+    # error::DeprecationWarning
     error::PendingDeprecationWarning
```

Comment:
welp seems like you have to do this one too...

Reply:
Sorry for chiming in. Please only ignore specific DeprecationWarnings. This can be achieved with a pattern, like [here](https://github.com/adfinis-sygroup/mySAGW/blob/main/api/setup.cfg#L44).

Reply:
much better suggestion!